docs/integrations/webhooks/events/index.mdx

```diff
@@ -475,7 +475,7 @@ Consult the [notifications section of the Channel Webhooks Guide](/docs/integrat
 
 | Name / Scope             | Description |
 |:-------------------------|:------------|
-| store/option/updated | Fires when you edit attributes for a local or shared variant option, including its display name and option values. Updates to the global store and channel locale as an override triggers the webhook.<br /><br />The `context` fields are present for only updates to overrides, not the global store. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview. |
+| store/option/updated | Fires when you edit attributes for a local or shared variant option, including its display name and option values. Updates to the channel locale as an override, not the global store, trigger the webhook.<br /><br />The `context` fields are present when you update the overrides. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview. |
```

Comment:
Same with this one:

`Fires when you customize display name and values for a local or shared variant option in a context of a channel and locale. For information on updating overrides, see the [International Enhancements for Multi-Storefront](/docs/store-operations/catalog/msf-international-enhancements) overview.`